sdkproject/Assets/Mapbox/Unity/Editor/PropertyDrawers/CoreVectorLayerPropertiesDrawer.cs

```diff
@@ -47,9 +47,9 @@ public override float GetPropertyHeight(SerializedProperty property, GUIContent
 			var sourceTypeProperty = property.FindPropertyRelative("geometryType");
 
 			float height = 0.0f;
-			height += (((((VectorPrimitiveType)sourceTypeProperty.enumValueIndex == VectorPrimitiveType.Line)) ? 6.0f : 5.0f) * EditorGUIUtility.singleLineHeight);
+			height += (((((VectorPrimitiveType)sourceTypeProperty.enumValueIndex == VectorPrimitiveType.Line)) ? 5.0f : 4.0f) * EditorGUIUtility.singleLineHeight);
```

Comment:
Where do these float values come from? Can we make this less opaque?

Reply:
These are the number of rows required for the UI based on the primitive. We can add a variable or code comment